js/main.js

```diff
@@ -0,0 +1,9 @@
+function getRandomCoordinate (min, max, decimal) {
```

Comment:
Здесь мы возвращаем дробное число, а не коордитану - следует отразить это в нейминге getRandomFloat как вариант
```suggestion
function getRandomFloat(min, max, decimal) {
```